### PR TITLE
Fix TS errors for GRAFBASE_API_KEY env var

### PR DESCRIPTION
**Pull request by mlafeldt**
Which might be undefined, but we require it.

# Type of change

- [ ] 💔 Breaking
- [ ] 🚀 Feature
- [x] 🐛 Fix
- [ ] 🛠️ Tooling
- [ ] 🔨 Refactoring
- [ ] 🧪 Test
- [ ] 📦 Dependency
- [ ] 📖 Requires documentation update

### CODE REVIEW

File: examples/nextjs/src/utils/server-side-fetch.ts [hunk 1]
```diff
@@ -1,10 +1,9 @@
 const serverSideFetch = async (query: string) =>
-  await fetch(process.env.GRAFBASE_API_URL as string, {
+  await fetch(process.env.GRAFBASE_API_URL!, {
     method: 'POST',
-    // @ts-ignore
     headers: {
       'Content-Type': 'application/json',
-      'x-api-key': process.env.GRAFBASE_API_KEY
+      'x-api-key': process.env.GRAFBASE_API_KEY!
```
Comment (fbjork) [inline examples/nextjs/src/utils/server-side-fetch.ts:6]: How will this work for local dev?

Comment (yoav-lavi) [reply examples/nextjs/src/utils/server-side-fetch.ts:6]: It's not needed in local dev AFAIK

**Review by vitalikda — APPROVED** [review]

File: examples/nextjs/src/utils/server-side-fetch.ts (hunk 1 above)
Comment (vitalikda) [reply examples/nextjs/src/utils/server-side-fetch.ts:6]: Yes, you don't need it, just set `GRAFBASE_API_URL=http://localhost:4000` in .env file

Comment (mlafeldt) [reply examples/nextjs/src/utils/server-side-fetch.ts:6]: The examples will set the API key to an empty string via the env file, e.g. https://github.com/grafbase/grafbase/blob/main/examples/nextjs/.env.example#L1

So this should work OOTB.

**Pull request merged**